### DISCUSSION TITLE
Add option to change generated commit message language

### DISCUSSION THREAD

**Discussion by RodrigoAroeira** [community/community#75766] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

I saw that even when changing the IDE language, the generated commit message will always be written in english and never in the selected language. Github Copilot Chat already has that option, so I think it would also be a good idea to do with the commit message, so they can be coherent within the repositories

**Comment by cannap** [comment]
yes thi would be awesome

**Comment by spenserblack** [comment]
What about if we could change the "templates" for the generated commit messages? For example, given that `Create {{ filename }}` is the default message for creating a new file (e.g. `Create README.md`), we could either change the wording (e.g. `Add {{ filename }}`) and/or change the language (`{{ filename }}を作った`).
